Exoft.Gamification.Api.Common/Models/NewUserModel.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Exoft.Gamification.Api.Common.Models
+{
+    public class NewUserModel
+    {
+        public string UserName { get; set; }
```

Comment:
Not sure, but I saw Login somewhere.... please check it... And keep everything consistent